test/routes/api/auth.js

```diff
@@ -0,0 +1,69 @@
+const dotenv = require("dotenv");
+dotenv.config();
+const mongoose = require("mongoose");
+let chai = require("chai"),
+  expect = chai.expect;
+var chaiHttp = require("chai-http");
```

Comment:
Any purpose of using var here?
```suggestion
let chaiHttp = require("chai-http");
```

Reply:
done